### ISSUE TITLE
Provide structure of app development

### ISSUE THREAD

**Issue by codingwithelijah** [amosproj/amos2022ss05-find-my-hearing-aid#21]
## User story
1. As a SD
2. I want / need provide a structure of the development process of an app
3. So that the POs know what (technical) parts to consider

## Acceptance criteria
* Meet with SDs to discuss approach to app development
* Agree on approach model and create a visualization

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
